modules/core/src/test/java/org/apache/ignite/util/GridCommandHandlerAbstractTest.java

```diff
@@ -288,4 +324,21 @@ protected void checkUserFutures() {
                 fail("Some transaction are not finished");
         }
     }
+
+    /**
+     * Creates default cache and preload some data entries.
+     *
+     * @param ignite Ignite.
+     * @param countEntries Count of entries.
+     */
+    protected void createCacheAndPreload(Ignite ignite, int countEntries) {
```

Comment:
I think you should not overload and leave it as it is, because in many places you need your own additional parameters for caches.

- org.apache.ignite.util.GridCommandHandlerTest # testKillHangingLocalTransactions need in setAtomicityMode,setWriteSynchronizationMode

- org.apache.ignite.util.GridCommandHandlerTest#testActiveTransactions - no need in affinity

- org.apache.ignite.util.GridCommandHandlerTest#testCacheIdleVerifyDumpForCorruptedDataOnNonePersistenceClientCache need in setDataRegionName

Etc.

Reply:
It's a code deduplication. Please do this.